aries-site/src/themes/aries.js

```diff
@@ -30,8 +30,14 @@ export const aries = deepMerge(hpe, {
       // background: undefined,
       border: { side: 'bottom' },
       color: 'text-strong',
-      extend: ({ column, sort, sortable }) => {
+      extend: ({ column, sort, sortable, theme }) => {
         return `
+          ${sort && sort.property === column && `
+            background: ${
+              theme.global.colors['background-contrast'][
```

Comment:
Nit picky but should we use `active-background` (same hex value, but feels like this is an "active" button)

Reply:
Good call. Will do.